### DISCUSSION TITLE
mlops/2021/02/07/python-streamlit-dashboard/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#130] [Comments]
#  Python Streamlit 사용법 - 프로토타입 만들기

https://zzsza.github.io/mlops/2021/02/07/python-streamlit-dashboard/

**Comment by zzsza** [comment]
<p>좋은자료 감사합니다!</p>

>_This comment was made by Disqus. 2021-02-09 11:16:28 김용주_

**Comment by zzsza** [comment]
<p>좋은 자료 잘봤습니다. :)</p>

>_This comment was made by Disqus. 2021-04-09 10:23:40 dskim_

**Comment by zzsza** [comment]
<p>앙 조아</p>

>_This comment was made by Disqus. 2021-05-31 08:48:13 Marco You_

**Comment by zzsza** [comment]
<p>streamlit 궁금했는데 잘 정리해주셔서 감사합니다.</p>

>_This comment was made by Disqus. 2021-10-29 08:11:36 triptojoon_

**Comment by theangkko** [comment]
좋은 자료 잘 봤습니다

**Comment by kjh75** [comment]
질문1. ***.py로 작성되는 파일은 어느 경로(폴더)에 저장해야 되나요?
질문2. streamlit run ***.py라는 명령어는 어디서 입력하나요?
저는 cmd창에서 env만들었고 ***.py는 메모장에서 작성하였습니다.
cmd창에서 streamlit run ***.py 를 어떻게 실행해도 Error: Invalid value: File does not exist: firstApp.py가 뜹니다.  아나콘다나 vscode없이는 streamlit 실행할 수 없는 건가요?
